### ISSUE TITLE
Crash when computer is put to sleep.

### ISSUE THREAD

**Issue by Tehelee** [Ikeiwa/EditSaber#31]
Your render thread times out when the computer is asleep, please account for this and pause execution or implement a recovery on wake.

[https://i.imgur.com/JuBi0KD.png](url)

**Comment by Ikeiwa** [comment]
I'm sorry but I don't think you can edit this with ue4 (or I need to compile the engine myself and I don't have the time) and this should be already implemented in ue4 by default (on my side this crash doesn't occur). So this "can" be a rare bug or a problem on your side (I can be wrong, I'm lacking skills in low level ue4 coding).

**Comment by Tehelee** [comment]
Thanks Ikeiwa for the quick reply, it looks like pausing the game and putting the computer to sleep is the issue. Is there any way you can rework it to not use pausing? The menu doesn't crash but in-edit it does.

https://issues.unrealengine.com/issue/UE-52315

Obviously we'd all love to see copy-paste implemented before fixing something like this.

**Comment by Ikeiwa** [comment]
Yeah, this is not my priority (right now I'm recoding  the editor almost from scratch) but this is still important, also I never use pausing in the editor so it's not the source of the crash.
